packages/web/src/errors/errors-list.ts

```diff
@@ -667,6 +667,18 @@ Use 'deployer.isDefined(name)' to check if the name is already used.
 Use 'deployer.isDefined(name)' to check if the name is already used.
 `,
 	},
+	RUN_FILE_NOT_FOUND_WITH_SUGGESTION: {
+		number: 617,
+		message: "Script not found. Please check the format: yarn algob run script.js --arg '<JSON String>'",
+		title: "Run file not found.",
+		description: "Script not found. Please check the format: yarn algob run script.js --arg '<JSON String>'",
+	},
+	RUN_ARGUMENT_INVALID: {
+		number: 618,
+		message: "The JSON string passed is invalid.",
```

Comment:
You could add parameter to this message so that user can have better understand
```suggestion
		message: "The JSON string passed %jsonString% is invalid.",
```